src/odemis/driver/pigcs.py

```diff
@@ -2838,6 +2844,7 @@ def __init__(self, name, role, port, axes, baudrate=38400,
         ac_to_axis = {} # address, channel -> axis name
         controllers = {} # address -> kwargs (axes, dist_to_steps, min_dist, vpms...)
         for axis, (add, channel, isCL) in axes.items():
+            channel = bytes(channel)
```

Comment:
Yes, that should help to keep the current compatibility. To be Python3 compatible, as channel might be a str (aka unicode), you'd need to write:
```
if isinstance(channel, int):
   channel = str(channel)
channel = channel.encode("ascii") # bytes
```